pkg/services/webhook.go

```diff
@@ -80,13 +80,17 @@ type BasicAuth struct {
 }
 
 type WebhookOptions struct {
-	URL                string        `json:"url"`
-	Headers            []Header      `json:"headers"`
-	BasicAuth          *BasicAuth    `json:"basicAuth"`
-	InsecureSkipVerify bool          `json:"insecureSkipVerify"`
-	RetryWaitMin       time.Duration `json:"retryWaitMin"`
-	RetryWaitMax       time.Duration `json:"retryWaitMax"`
-	RetryMax           int           `json:"retryMax"`
+	URL                 string        `json:"url"`
+	Headers             []Header      `json:"headers"`
+	BasicAuth           *BasicAuth    `json:"basicAuth"`
+	RetryWaitMin        time.Duration `json:"retryWaitMin"`
+	RetryWaitMax        time.Duration `json:"retryWaitMax"`
+	RetryMax            int           `json:"retryMax"`
+	InsecureSkipVerify  bool          `json:"insecureSkipVerify"`
+	MaxIdleConns        int           `json:"maxIdleConns"`
+	MaxIdleConnsPerHost int           `json:"maxIdleConnsPerHost"`
+	MaxConnsPerHost     int           `json:"maxConnsPerHost"`
+	IdleConnTimeout     time.Duration `json:"idleConnTimeout"`
```

Comment:
We can't use time.Duration for parsing time values - we will need to use a string and then parse the actual `time.Duration` by doing something like this 

```golang
	idleConnTimeout, err := time.ParseDuration(s.opts.IdleConnTimeout)
	if err != nil {
		return errors.Wrap(err, "failed to parse idle connection timeout")
	}
```

and then passing in `idleConnTimeout` to the `NewTransport` function.